test/e2e/network/ingressclass.go

```diff
@@ -42,7 +43,7 @@ var _ = SIGDescribe("IngressClass [Feature:Ingress]", func() {
 		cs = f.ClientSet
 	})
 
-	ginkgo.It("should set default value on new IngressClass", func() {
+	framework.ConformanceIt("should set default value on new IngressClass", func() {
 		ingressClass1, err := createIngressClass(cs, "ingressclass1", true, f.UniqueName)
```

Comment:
this assumes there is no default ingressclass present in the cluster already... not sure it is valid for an e2e to make assumptions about the existing cluster config, and almost certainly not valid for a conformance test to do so... that would mean a cluster that installs its own default ingress class could not be conformant, which doesn't seem right

Reply:
also, this test can't be run in parallel with the other test in this suite... this needs to be resolved before promotion to conformance